src/nxdoc/web-ui/web-ui-upgrade-notes/web-ui-11-1-upgrade-notes.md

```diff
@@ -79,5 +85,95 @@ Removed variables can be safely deleted from your themes or elements. Deprecated
+| documentHistory.category      | audit.category      |
+| documentHistory.comment       | audit.comment       |
+| documentHistory.date          | audit.date          |
+| documentHistory.filter.after  | audit.filter.after  |
+| documentHistory.filter.before | audit.filter.before |
+
+### Reference to invalid packages in the project
+
+The reference to packages that no longer exist, such as `nuxeo-dam` or `nuxeo-spreadsheet`, or the presence of JSF specific contributions, might cause conflicts and prevent a project from working properly.
+
+{{!--     ### nx_asset ###
+    path: /default-domain/workspaces/Product Management/Documentation/Documentation Screenshots/NXDOC/Master/Web UI 11.x upgrade notes/Package removal in Studio's Application Definition
+    name: app-def-package-removal.png
+    1.1.3#screenshot#up_to_date
+--}}
+![Package removal in Studio's Application Definition](nx_asset://b47d5c8b-c1ba-4d2a-ab0b-8b8378d73c8d ?w=650,border=true)
+
+#### Nuxeo DAM addon removal
+
+Nuxeo DAM no longer exists as an addon and its contributions are now default on Web UI. If you're using Nuxeo Studio, you will see the Nuxeo DAM in the removed addons list when upgrading your project through the application definition page.
+
+#### Spreadsheet addon removal
+
+Spreadsheet addon is now loaded by default but the button contribution is disabled. Users can rely on Studio Designer to re-enable the `spreadSheet` *button* in the `RESULTS_ACTIONS` slot (see [WEBUI-90](https://jira.nuxeo.com/browse/WEBUI-90)).
+
+{{!--     ### nx_asset ###
+    path: /default-domain/workspaces/Product Management/Documentation/Documentation Screenshots/NXDOC/Master/Web UI 11.x upgrade notes/Spreadsheet package enable button in Designer
+    name: spreadsheet-package-enable-button-designer.png
+    1.1.3#screenshot#up_to_date
+--}}
+![Spreadsheet package enable button in Designer](nx_asset://b8fd28dd-0272-43b1-a083-dfde295c312b ?w=650,border=true)
+
+### BROWSER_ACTIONS slot removal
+
+The `BROWSER_ACTIONS` nuxeo slot was removed under [NXP-26184](https://jira.nuxeo.com/browse/NXP-26184). It was already deprecated since Web UI 0.9 and had no known usage. It was replaced by the `RESULTS_SELECTION_ACTIONS` slot.
+
+### Forked nuxeo-document-content might lose selection actions
+
+After [NXP-25345](https://jira.nuxeo.com/browse/NXP-25345), Nuxeo Web UI introduced the ability to override selection actions. Elements that were forked from an older version of `nuxeo-document-content` and that override the `selectionActions` native slot with new content will be missing the contributions to the `RESULTS_SELECTION_ACTIONS` nuxeo slot. This can be rectified by adding the desired actions to the new slot, and by deleting the following piece of code:
+
+```
+<div slot=“selectionActions”>
+  <nuxeo-slot slot=“BROWSE_ACTIONS” model=“[[browseActionContext]]“></nuxeo-slot>
+</div>
+```
+
+### Picture document page being displayed on doctypes using the Picture facet
```

Comment:
I think we should have a `### Breaking changes`, and place this and the two above under it. Something like:

```
### Breaking changes
...
#### BROWSER_ACTIONS slot removal
...
#### Forked nuxeo-document-content might lose selection actions
...
#### Picture document page being displayed on doctypes using the Picture facet
...
```

Reply:
You're right. I didn't include the `### Breaking changes` from your notes because I thought we didn't want to advertise them as breaking changes, but thinking about it again, if they are really breaking changes we should state it.